src/test/java/unit/GcpNotebookControlled.java

```diff
@@ -284,9 +283,7 @@ static void assertNotebookState(String resourceName, String notebookState, UUID
                 "--name=" + resourceName,
                 "--workspace=" + workspaceId);
     assertEquals(notebookState, describeNotebook.state, "notebook state matches");
-    if (!notebookState.equals("PROVISIONING")) {
```

Comment:
can you keep this? since proxy url shouldn't exist for PROVISIONING state.

Reply:
Oops, done